components/button/stories/button.stories.js

```diff
@@ -184,3 +212,11 @@ WithForcedColors.parameters = {
 WithForcedColors.args = {
 	iconName: "Actions",
 };
+
+export const Wrapping = CustomButton.bind({});
+Wrapping.args = {
+	showOneButtonPerLine: true,
```

Comment:
```suggestion
	layout: "stacked",
```

Reply:
`layout` is a much better name, thanks! I've made all these updates to rename that control and make it visible.